Library/Formula/stress.rb

```diff
@@ -1,13 +1,16 @@
-require 'formula'
-
 class Stress < Formula
-  homepage 'http://rpmfind.net/linux/RPM/fedora/18/x86_64/s/stress-1.0.4-8.fc18.x86_64.html'
-  url 'ftp://ftp.oregonstate.edu/.2/gentoo/distfiles/stress-1.0.4.tar.gz'
-  sha1 '7ccb6d76d27ddd54461a21411f2bc8491ba65168'
+  homepage "http://people.seas.harvard.edu/~apw/stress/"
+  url "http://people.seas.harvard.edu/~apw/stress/stress-1.0.4.tar.gz"
+  mirror "https://mirrors.kernel.org/debian/pool/main/s/stress/stress_1.0.4.orig.tar.gz"
+  sha256 "057e4fc2a7706411e1014bf172e4f94b63a12f18412378fca8684ca92408825b"
```

Comment:
The SHA1 is different from the old tarball, but this matches the SHAs in the Debian `.dsc` file so it seemed safe.